### ISSUE TITLE
Cannot move focus back into view "..." once navigated

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#117348]
Testing #116559

Tab all the way to the "..." in the explorer view top right and arrow left or right. Then tab out and try to tab back in, doesn't work.

**Comment by isidorn** [comment]
Good catch, I can repro
